content/filters/string_filters.md

```diff
@@ -0,0 +1,273 @@
+# String Filters
+
+String filters are used to manipulate outputs and variables of the [string](/) type.
```

Comment:
Yes. This sentence is copied from Shopify docs about liquid string filters.
Link removed.